alternator/executor.cc

```diff
@@ -895,9 +917,12 @@ static void verify_billing_mode(const rjson::value& request) {
         if (!rjson::find(request, "ProvisionedThroughput")) {
             throw api_error::validation("When BillingMode=PROVISIONED, ProvisionedThroughput must be specified.");
         }
+        const rjson::value& throughput = request["ProvisionedThroughput"];
```

Comment:
Please try to avoid searching for the same attribute multiple times. We already do rjson::find above so let's store it in a variable, something like
```c++
rjson::value *provisioned_throughput = rjson::find(request, "ProvisionedThroughput");
if (!provisioned_throughput) {
  ....
}
```

Reply:
You didn't answer this comment, it's still relevant.